Skype/UCWA/TraceAndDebugTools.md

```diff
@@ -1,4 +1,6 @@
-
+---
+title: Trace and debug tools Fiddler debugging proxy
```

Comment:
Remove "Fiddler debugging proxy" If you get a duplicate-title problem, add a final word "reference"

Reply:
added reference, I am almost certain without that , I'd get a warning for duplicates.